infrastructure/sandbox/PreProvisioner/lambda/deploy_terraform/fleet/templates/cronjobs.yaml

```diff
@@ -0,0 +1,357 @@
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  labels:
+    app: fleet
+    chart: fleet
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+  name: {{ .Values.fleetName }}
+  namespace: {{ .Release.Namespace }}
+spec:
+  schedule: {{ .Values.crons.vulnerabilities }}
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: {{ .Values.fleetName }}
+            imagePullPolicy: Always
+            command: [/usr/bin/fleet]
+            args: ["vulnerabilities"]
+            image: {{ .Values.imageRepo }}:{{ .Values.imageTag }}
+            ports:
+            - name: {{ .Values.fleetName }}
+              containerPort: {{ .Values.fleet.listenPort }}
+            resources:
+              limits:
+                cpu: {{ .Values.resources.limits.cpu }}
+                memory: {{ .Values.resources.limits.memory }}
+              requests:
+                cpu: {{ .Values.resources.requests.cpu }}
+                memory: {{ .Values.resources.requests.memory }}
```

Comment:
not sure what we are setting here, but in theory we can now set this value decently high, but the fleet server resource limits pretty conservative. Maybe in a follow up PR? Just thought I'd mention so we don't forget.

Reply:
I'll check in on the memory usage later and see where we can set it.